src/components/FormElements/PubKeyPicker.tsx

```diff
@@ -7,34 +7,33 @@ interface PubKeyPickerProps extends Omit<InputProps, "fieldSize"> {
   label: string;
   labelSuffix?: string | React.ReactNode;
   placeholder?: string;
+  readOnly?: boolean;
```

Comment:
I added this because muxed account result displays `pubkeypicker`, but with `readOnly`